src/Differ/DifferInterface.php

```diff
@@ -20,12 +20,11 @@ interface DifferInterface
     /**
      * Create diff.
      *
-     * @param string $old
-     * @param string $new
+     * @param string       $old
+     * @param string       $new
+     * @param \SplFileInfo $file
      *
      * @return string
-     *
-     * TODO: on 3.0 pass the file name (if applicable) for which this diff is
      */
-    public function diff($old, $new);
+    public function diff($old, $new, $file = null);
```

Comment:
I think we type in the signature itself as well? like:
>public function diff($old, $new, \SplFileInfo $file = null);

Reply:
👍